test/test_backends.py

```diff
@@ -445,7 +445,8 @@ def test_parse_hostspec(hostspec, expected):
     ('kubectl://pod', 'pod', None, None, None),
     ('kubectl://pod?namespace=n', 'pod', None, 'n', None),
     ('kubectl://pod?container=c&namespace=n', 'pod', 'c', 'n', None),
-    ('kubectl://pod?namespace=n&kubeconfig=k', 'pod', None, 'n', 'k')
+    ('kubectl://pod?namespace=n&kubeconfig=k', 'pod', None, 'n', 'k'),
+    ('kubectl://pod?context=ctx&container=c', 'pod', 'ctx', 'c', None)
```

Comment:
you have to add a "context" parameter in `pytest.mark.parametrize` and in the function arguments and then `assert backend.context == context`

Reply:
Ohh, just put in some updates once I understood how. I hadn't done the assertion bit. I'll take a look now.

Thanks for your speedy reply @philpep

Reply:
Done this time (I'll stick around to make sure the test passes)

Reply:
Just trying to tackle the linter :/

Reply:
@philpep I think the 80 char limit is a bit too narrow. When combined with the indentation rule, it's never ending trying to satisfy both constraints.

Are you able to suggest how I should layout the test? Or can we look at relaxing the rules (might lower developer burden - or maybe it's just me).